avocado/core/settings.py

```diff
@@ -250,6 +267,8 @@ def get_value(self, section, key, key_type=str, default=no_default,
         """
         try:
             val = self.config.get(section, key)
+        except ConfigParser.NoSectionError:
+            return self._handle_no_section(section, default)
```

Comment:
It's still the same exception, just a bit more accurate message, so I'm fine with the extra "bloat" :)